Ontology/Asset/Component/Valve/Valve.json

```diff
@@ -1,5 +1,5 @@
 {
-  "@id": "dtmi:digitaltwins:rec_3_3:asset:Valve;1",
```

Comment:
Do we still have the old version as ;1 in the repo?

Reply:
From my point of view it would be a lot easier if we could keep those older versions in the git history, rather than keep them around as older versions in the current HEAD, especially while we are still on the OWL->DTDL translation pipeline.

If they absolutely need to stick around, perhaps they could be placed in a separate directory structure ("Deprecated models")?

Reply:
I like having a separate directory for Deprecated models for now, but that's probably just because I'm not very proficient with Git to be able to easily get at the history.